### PR TITLE
Feature/push to wiki

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
Because manually pushing the docs to the wiki takes time and effort

### What Changed?
Added auto-push config to gem
Added documentation on auto-push to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,3 +158,27 @@ end
...
+
+**To enable this feature**
+  1. Clone your wiki repo from Github into the same parent folder as your app
+
+  Your folder structure should be
+
+    ../projects/my_rails_app
+
+    ../projects/my_rails_app.wiki
+
```
Comment (albaer) [inline README.md]: Nice documentation here.

File: README.md [hunk 2]
```diff
@@ -157,3 +158,27 @@ end
...
+  2. Set auto_push to true in `spec_helper.rb` or `test_helper.rb`
+
+    ``` ruby
+      SmashingDocs.config do |c|
+        # configs
+        c.auto_push = true
```
Comment (albaer) [inline README.md]: :heart: configs

File: gem_rspec/base_spec.rb [hunk 3]
```diff
@@ -115,4 +115,59 @@
...
+        expect(SmashingDocs.current.send(:wiki_repo_exists?, wiki_repo)).to eq(false)
+      end
+    end
+  end
+
+  describe "#copy_docs_to_wiki_repo(wiki_repo)" do
+    let(:wiki_repo) { "../#{SmashingDocs.current.send(:app_name)}.wiki" }
+    let(:output_file) { SmashingDocs.current.send(:output_file) }
+    it "makes a copy of the docs in the wiki folder" do
+      `rm -f ../smashing_docs.wiki/"#{output_file}"` # -f just in case the file's stubborn
```
Comment (albaer) [inline gem_rspec/base_spec.rb]: The idea of something that goes in and actually deletes stuff off of your computer is kind of terrifying.

**Comment by albaer** [comment]
Looks good to me! The interpolated rm -f is a little scary since in theory it could be "../../../whatever/" and end up deleting something important, but that seems like a pretty paranoid edge case, so I say :ship: it.

**Comment by TylerRockwell** [comment]
I would change that if it was in the app, and not the tests. Just remember to never set your config file to "../../../../my_important_file" and run the test suite.

**Comment by TylerRockwell** [comment]
Thanks for the review, @albaer !
